addons/rook/1.7.11/install.sh

```diff
@@ -237,9 +237,16 @@ function rook_cluster_deploy_upgrade() {
 
     # 4. https://rook.io/docs/rook/v1.6/ceph-upgrade.html#4-wait-for-the-upgrade-to-complete
     echo "Awaiting rook-ceph operator"
-    if ! $DIR/bin/kurl rook wait-for-rook-version "$ROOK_VERSION" --timeout=1200 ; then
-        logWarn "Rook version not yet rolled out"
-        kubectl -n rook-ceph get deployment -l rook_cluster=rook-ceph -o jsonpath='{range .items[*]}{.metadata.name}{"  \treq/upd/avl: "}{.spec.replicas}{"/"}{.status.updatedReplicas}{"/"}{.status.readyReplicas}{"  \trook-version="}{.metadata.labels.rook-version}{"\n"}{end}'
```

Comment:
in my opinion this was correct the way it was. The version is not yet rolled out because there is either multiple rook versions or the replicas are not ready and up to date which is the same check that the `kurl rook wait-for-rook-version` validates. The proposed change will hide the output if all versions are the same but the replicas are not ready and up to date. This seems like a regression.

The correct fix in my opinion would be to backport this code to versions 1.4.9 - 1.6.11

Reply:
Hi @emosbaugh,

I think I understand your comment. It is helpful we get the output from the rook replicas and versions every time that we face an timeout. We should not only WARNING saying that we found more than one version when not more the one version was found.